bapsflib/_hdf_mappers/map_digitizers/siscrate.py

```diff
@@ -453,8 +453,8 @@ def construct_dataset_name(self, board, channel,
                     found += 1
 
             if found == 1:
-                warn_str = ('\nconfig_name not specified, '
-                            'assuming ' + config_name + '.')
+                warn_str += ('\n** Warning: config_name not specified,'
```

Comment:
remove `'** Warnings:'` from `warn_str`. Since Python's warning system is used, this is addition is redundant.